templates/news/news-fr.html

```diff
@@ -1,13 +1,228 @@
 ---
 {
-	"title": "Page de contenu",
-	"language":	"fr",
+	"title": "Nouvelles",
+	"language": "fr",
 	"altLangPage": "news-en.html",
 	"breadcrumbs": false,
 	"secondlevel": false,
-	"dateModified": "2016-07-20",
+	"dateModified": "2023-02-21",
 	"share": "true",
 	"pageType": "news"
 }
 ---
-{{>placeholdercontent}}
+<div lang="en">
+	<div class="mrgn-bttm-md" data-wb-randomize='{ "shuffle": true, "selector": "a", "toggle": "hide" }' id="wb-auto-4">
+		<a href="#" class="hide">
+			<figure>
+				<img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/F5F5F5/000000.png" alt="">
+				<figcaption class="well brdr-rds-0">
+					<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
+				</figcaption>
+			</figure>
+		</a>
+		<a href="#" class="hide">
+			<figure><img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/26374A/FFFFFF.png" alt="">
+				<figcaption class="well brdr-rds-0">
+					<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
+				</figcaption>
+			</figure>
+		</a>
+		<a href="#" class="hide">
+			<figure>
+				<img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/7834BC/FFFFFF.png" alt="">
+				<figcaption class="well brdr-rds-0">
+					<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
+				</figcaption>
+			</figure>
+		</a>
+		<a href="#" class="hide">
+			<figure>
+				<img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/0000FF/000000.png" alt="">
+				<figcaption class="well brdr-rds-0">
+					<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
+				</figcaption>
+			</figure>
+		</a>
+	</div>
+	<div class="clearfix">
+		<div class="pull-right">
+			{% include components/gc-follow-us/gc-follow-us.html isInline=true %}
+		</div>
+	</div>
+	{% include components/gc-features/features.html %}
+	<div class="row mrgn-tp-md">
```

Comment:
Ajoute le language attribute

```suggestion
	<div class="row mrgn-tp-md" lang="en">
```